app/src/main/res/layout/card_release_info.xml

```diff
@@ -1,77 +1,82 @@
 <?xml version="1.0" encoding="utf-8"?>
-
-<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Let's use a ConstraintLayout :). Also, can we get rid of the LinearLayout inside it?

Reply:
Done boss